tripy/docs/conf.py

```diff
@@ -178,40 +187,40 @@ def process_docstring(app, what, name, obj, options, lines):
                     ":returns:" in doc
                 ), f"For: {obj}, return value is not documented. Please ensure you've included a `Returns:` section"
 
-    # new docstring logic:
-    # first figure out if we should it is the new docstring
+    # New docstring logic:
+    # First figure out if we should it is the new docstring.
     if name.split(".")[-1] in TYPE_VERIFICATION.keys():
         cleaned_name = name.split(".")[-1]
         add_text_index = -1
         for index, block in enumerate(blocks):
             if re.search(r".. code-block::", block):
-                type_dict = TYPE_VERIFICATION[cleaned_name][1]["types"]
+                type_dict = TYPE_VERIFICATION[cleaned_name].dtypes
                 blocks.insert(index, "Type Constraints:")
                 index += 1
+                # Add the dtype constraint name and the dtypes that correlate. Remove excluded dtypes.
                 for type_name, dt in type_dict.items():
-                    blocks.insert(index, f"    - {type_name}: " + ", ".join(dt))
+                    blocks.insert(
+                        index,
+                        f"    - **{type_name}**: :class:`" + "`, :class:`".join(set(dt) - {"int4", "float8"}) + "`",
+                    )
                     index += 1
                 blocks.insert(index, "\n")
                 break
             if re.search(r":param \w+: ", block):
-                add_text_index = re.search(r":param \w+: ", block).span()[1]
                 param_name = re.match(r":param (\w+): ", block).group(1)
```

Comment:
How does this work for multiple parameters?

Reply:
It goes line by line through the docstring so there will never be two parameters on the same line.